### PR TITLE
[WebAPI] Add 7 presentationconnection test cases

### PR DESCRIPTION
**Pull request by yhe39**
Test results: under crosswalk for Android 20.50.532.0
PresentationConnection_basic-manual.html                pass
PresentationConnection_onclose-manual.html              pass
PresentationConnection_onconnect-manual.html            fail
PresentationConnection_onterminate-manual.html          fail
PresentationConnection_state_closed-manual.html         fail
PresentationConnection_state_connected-manual.html      pass
PresentationConnection_state_terminated-manual.html     fail

### CODE REVIEW

**Comment by yhe39** [comment]
@Honry @zqzhang PTAL. thanks a lot!

File: presentation/PresentationConnection_basic-manual.html [hunk 1]
```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<!--
```
Comment (zqzhang) [inline presentation/PresentationConnection_basic-manual.html:2]: Please remove all license information from test files before submitting to W3C.

All W3C tests are licensed under http://www.w3.org/Consortium/Legal/2008/04-testsuite-copyright.html, see https://github.com/w3c/web-platform-tests/blob/master/LICENSE

File: presentation/PresentationConnection_basic-manual.html [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,
+EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+<meta charset="utf-8">
+<title>PresentationConnection basic attributes for Controlling User Agent</title>
```
Comment (zqzhang) [inline presentation/PresentationConnection_basic-manual.html:30]: These checkpoints have been covered by http://w3c-test.org/presentation-api/controlling-ua/idlharness.html

File: presentation/PresentationConnection_basic-manual.html [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+</div>
+<script type='text/javascript'>
+  var connection = null;
+  var url = "display/presentation.html";
+  function onclickStart() {
+    var request = new PresentationRequest(url);
+    request.start().then(function(conn) {
+      connection = conn;
+      document.getElementById("result").innerHTML = "start success" + "<br/>"; 
+      if(conn instanceof PresentationConnection) {
```
Comment (zqzhang) [inline presentation/PresentationConnection_basic-manual.html:62]: Actually why not automate these tests using testharness.js assertions?

**Comment by zqzhang** [comment]
> PresentationConnection_onclose-manual.html pass
> PresentationConnection_onconnect-manual.html fail
> PresentationConnection_onterminate-manual.html fail
> PresentationConnection_state_closed-manual.html fail
> PresentationConnection_state_connected-manual.html pass
> PresentationConnection_state_terminated-manual.html fail

Why not combine these test cases into one test file? They seem have many duplicated code. Please also think of automating them.

File: presentation/PresentationConnection_basic-manual.html (hunk 3 above)
Comment (yhe39) [inline presentation/PresentationConnection_basic-manual.html:62]: As under testharness, the presentation API start() will encounter “requires user gesture” error (Louay figured out this problem in the meeting W3C Second Screen F2F).
So these test cases are defined as manual.

File: presentation/PresentationConnection_onclose-manual.html [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+  </ol>
+  <div align="middle">
+    <input type='button' value='start' onclick="onclickStart()" />
+  </div>
+  <div align="middle">
+    <input type='button' value='close' onclick="onclickClose()" />
+  </div>
+  <div id="result" style="width:100%; height:100px; word-wrap: break-word; word-break: normal;"></div>
+</div>
+<script type='text/javascript'>
```
Comment (zqzhang) [inline presentation/PresentationConnection_onclose-manual.html:54]: No need `type='text/javascript'`

**Pull request closed without merging**